makefile

```diff
@@ -93,7 +94,7 @@ dep-add: build/containerpilot_build
 # run 'GOOS=darwin make tools' if you're installing on MacOS
 ## set up local dev environment
 tools:
-	@go version | grep 1.8 || (echo 'go1.8 not installed'; exit 1)
+	@go version | grep 1.8 || (echo 'go1.8 should be installed')
```

Comment:
Ok, that's fair so long as we're pinning for the build process. The "casual contributor" case I think we're covered for with the Docker builder. I can't think of any other impact, so LGTM 👍